modules/ossm-upgrading-operator.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/upgrading-ossm.adoc
+
+[id="ossm-upgrading-operator_{context}"]
+= Upgrading the Operators
+
+[IMPORTANT]
+====
+The version of the Operator does *not* determine the version of your service mesh.  The current Operator supports both v1 and v2 service meshes.
+
+Updating the Operator does not affect the version of any component other than the Operator.  Updating the Operators does _not_ update the `ServiceMeshControlPlane` version or  deployments.
+====
+
+When you installed your Operators, you selected an *Update Channel* and an *Approval Strategy*.  Those two settings determine when and how your Operators are updated.
+
+.Interaction of Update Channel and Approval Strategy
+[options="header"]
+[cols="a, a, a"]
+|====
+| |Versioned channel|"Stable" or "Preview" Channel
+|*Automatic*
+|Automatically updates Operator for minor and patch releases for that version only.  Will not automatically update to the next major version (that is, from version 2.0 to 3.0). Manual change to Operator subscription required to update to the next major version.
```

Comment:
Automaticlly updates control plane for minor and patch releases

Reply:
It's my understanding that the Operator does NOT automatically upgrade the SMCP (which is what customers are worried about, accidentally upgrading their systems when the Operator updates).

Reply:
sorry, yeah that's right, I corrected the sentence- but I'm still confused by this part. There's two sentences that contradict each other. In this one, it says "Automatically updates operator for minor and patch releases", in the next sentence below it says "Automatically updates Operator for Major, minor and patch releases".

We update the operator to the latest version always if you set this flag. Updating the operator then triggers an automatic control plane update for patch versions

Reply:
I'm guessing you might be looking at the file and not the preview?  These two sentences are part of a table that shows the interaction between selecting an upgrade channel (stable or stable 2.1) and a manual or automatic approval process for updates.

Reply:
that's right- my fault for not looking at the actual preview. I didn't think we're doing version channels, but I haven't looked at it in a while